### DISCUSSION TITLE
Options to input precise values for slide bars.

### DISCUSSION THREAD

**Discussion by ppgamer** [ppy/osu#24900] [Ideas]
Currently, there is no easy way to make values in slide bars (eg. mouse sensitivity, mod difficulty adjust, etc.) be precisely the value that you want, so I propose two options to help this issue:

1. Add a keybind that locks the increment/decrement of the value to be the multiple of 0.05:
Perhaps, when holding shift while changing the value, the mouse sensitivity will be locked to a certain value that isn't affected by the mouse sensitivity, and the value will be locked to a multiple of 0.05, (aka it can only move up/down by 0.05) so it's easier to get a precise number.
![image](https://github.com/ppy/osu/assets/12120002/58354079-9690-4da0-9e23-c744b1514945)
I imagine it would be similar to slider angle snap in the editor

2. Add a method where users can simply type out their value:
Perhaps, when you double-click on the slider's header, a small pop-up appears that lets you input the value that you want into the box.
![image](https://github.com/ppy/osu/assets/12120002/df79c4fe-b89e-4e7f-b60d-b2a80aedddfd)
I imagine it would be similar to slider velocity popup in the editor

**Comment by peppy** [comment]
It's planned to add this everywhere in the interface eventually.

**Comment by Number-3434** [comment]
How's it going? I saw a [video with this feature implemented](https://youtu.be/Kqu4TUOO5IY?t=379), but I'm yet to see it implmented in the settings.

**Reply by peppy** [reply]
Which slider bar are you wanting to see this on? It's already been implemented on the ones in the video you linked, but we haven't deployed the new design globally yet.

**Reply by Number-3434** [reply]
I'd like to see it on all slider bars if possible, especially the ones in the settings.

**Reply by peppy** [reply]
This will eventually happen when we bring all UI elements up-to-date with the latest design one (which you can see in the editor).

**Reply by 2710165659** [reply]
> Which slider bar are you wanting to see this on? It's already been implemented on the ones in the video you linked, but we haven't deployed the new design globally yet.

If possible, could this also be applied to mod parameter adjustments?
After the 2026.326.0-tachyon update, the AC mod supports 0.1 precision, but it’s very difficult to adjust to an exact value using only the slider.
